app/sdk/utils/branding/BrandingManager.java

```diff
@@ -0,0 +1,126 @@
+package sdk.utils.branding;
+
+import akka.stream.Materializer;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.net.util.Base64;
+import org.apache.http.client.utils.URIBuilder;
+import org.asynchttpclient.AsyncHttpClientConfig;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import play.api.libs.ws.WSClientConfig;
+import play.api.libs.ws.ahc.AhcConfigBuilder;
+import play.api.libs.ws.ahc.AhcWSClientConfig;
+import play.api.libs.ws.ahc.AhcWSClientConfigFactory;
+import play.api.libs.ws.ssl.SSLConfigFactory;
+import play.libs.ws.WSClient;
+import play.libs.ws.WSRequest;
+import play.mvc.Http;
+import scala.concurrent.duration.Duration;
+import sdk.utils.JsonUtils;
+
+import javax.inject.Inject;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Author: Karis Sponsler
+ * Date: 11/22/17
+ */
+public class BrandingManager {
+    private static WSClient wsClient;
+    @Inject
+    private static Materializer materializer;
+
+    public static CompletionStage<Branding> getBranding(String appId, String version, String os) {
+        WSRequest request;
+        String routerUrl = "https://go-monolith-v2-sbx.herokuapp.com";
```

Comment:
The router URL should be parameterized, or otherwise configurable

Reply:
addressed.